beacon-chain/sync/validate_beacon_blocks.go

```diff
@@ -262,6 +273,22 @@ func (s *Service) validateBeaconBlock(ctx context.Context, blk interfaces.ReadOn
 	return nil
 }
 
+func validateDenebBeaconBlock(blk interfaces.ReadOnlyBeaconBlock) error {
+	if blk.Version() < version.Deneb {
+		return nil
+	}
+	commits, err := blk.Body().BlobKzgCommitments()
+	if err != nil {
+		return errors.New("unable to read commitments from deneb block")
+	}
+	// [REJECT] The length of KZG commitments is less than or equal to the limitation defined in Consensus Layer
+	// -- i.e. validate that len(body.signed_beacon_block.message.blob_kzg_commitments) <= MAX_BLOBS_PER_BLOCK
+	if len(commits) > fieldparams.MaxBlobsPerBlock {
+		return errRejectCommitmentLen
```

Comment:
can we return the length of the commits or log it

Reply:
will do